NAudio.Midi/Midi/MidiInSysexMessageEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace NAudio.Midi
+{
+    /// <summary>
+    /// MIDI In Sysex Message Information
+    /// </summary>
+    public class MidiInSysexMessageEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Create a new Sysex MIDI In Message EventArgs
+        /// </summary>
+        /// <param name="message"></param>
+        /// <param name="timestamp"></param>
```

Comment:
maybe add what units the timestamp is measured in to this comment

Reply:
or just delete the empty param comments as it's clear from the property below

Reply:
Done